graph/driver/driver.go

```diff
@@ -0,0 +1,43 @@
+package driver
+
+import (
+	"context"
+	"errors"
+
+	"github.com/ONSdigital/dp-code-list-api/models"
+)
+
+var ErrNotFound = errors.New("not found")
```

Comment:
its actually just so there's one static 'not found' response and err type that can be check by other apps, but i agree that error handling generally needs more thought and this might not live here in the long term